portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -37,60 +39,92 @@
   private static final String REFLECTION_PARAMETER = "reflection";
   private static final String COMMENT_PARAMETER = "Comment";
   private static final String TIMESTAMP_PARAMETER = "timestamp";
+  private static final String MAXCOMMENT_PARAMETER = "max-comment";
   private static final String APPLICATION_JSON_PARAMETER = "application/json;";
   private static final String COMMENT_HTML_PARAMETER = "/comments.html";
-
+  private static final String COMMENT_FORM = "comment-form";
+  private static final String MAX_FORM = "max-form";
 
   private String userNameInput;
   private String reflectionInput;
-  private long idGiven;
-  private long timestampOfComment;  
+  private String timestampOfComment;
```

Comment:
I would store this variable as a DateTime. Only convert it to String/Long when you need to (i.e. writing to the database, returning in json, etc.)

Reply:
LocalDateTime or Instant would be good here.